src/components/Link.js

```diff
@@ -25,8 +25,8 @@ class CSVLink extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const { data, headers, separator, uFEFF } = nextProps;
-    this.setState({ href: this.buildURI(data, uFEFF, headers, separator) });
+    const { data, headers, separator, uFEFF, enclosingCharacter } = nextProps;
```

Comment:
Hello @muhve  ,
Thank you for your contribution
componentWillReceiveProps has been deprecated. We have merged another PR (#195) which move the code to another method. could you please check this PR and update the code accordingly?